queue/test/fiskaltrust.Middleware.Storage.AcceptanceTest/AbstractQueueItemRepositoryTests.cs

```diff
@@ -192,6 +192,27 @@ public async Task InsertOrUpdateAsync_ShouldAddEntry_ToTheDatabase()
             insertedEntry.Should().BeEquivalentTo(entryToInsert);
         }
 
+        [Fact]
```

Comment:
Maybe we can add a few more tests to validate different datetime formats. I tend to be a bit paranoid when it comes to datetimes and conversions